server/data/data-jmap/src/main/java/org/apache/james/jmap/api/change/MailboxChangeRepository.java

```diff
@@ -0,0 +1,36 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.jmap.api.change;
+
+import java.util.Optional;
+
+import org.apache.james.jmap.api.change.MailboxChange.Limit;
+import org.apache.james.jmap.api.change.MailboxChange.State;
+import org.apache.james.jmap.api.vacation.AccountId;
```

Comment:
Why this import on vacation?

Should we promote that POJO outside of the vacation package to `org.apache.james.jmap.api.AccountId;` ?